helmfile.d/10-matrix.yaml

```diff
@@ -17,8 +17,20 @@ releases:
     macaroonSecretKey: {{ env "MACAROON_SECRET_KEY" | default "macaroon_secret" }}
     formSecret: {{ env "FORM_SECRET" | default "form_secret" }}
     signingKey: {{ env "SIGNING_KEY" | default "signing_key" | quote }}
+    antiscam: true
```

Comment:
The values in this helmfile are only useful for the integration test. Now that there are two different paths (with and without antiscam), maybe we could extend the test to cover both? Not related to this PR in any case, I would leave antiscam as undefined and maybe extend integration tests for antiscam on a followup.